generator/templates/liquiditymining.payloaddeployment.template.ts

```diff
@@ -0,0 +1,27 @@
+export const liquidityMiningPayloadDeploymentTemplate = (
+  poolChain: string,
+  testContractName: string,
+  scriptName: string
+) => {
+  return`
+  // SPDX-License-Identifier: MIT
+  pragma solidity ^0.8.0;
+
+import {${testContractName}, IPermissionedPayloadsController}
+  from './${testContractName}.t.sol';
+import {${poolChain}Script} from '../../lib/aave-address-book/lib/aave-v3-origin/lib/solidity-utils/src/contracts/utils/ScriptUtils.sol';
+
+contract ${scriptName} is ${testContractName}, ${poolChain}Script {
+  // solium-disable-next-line
+  function setUp() public override {}
+
+  // todo: specify address when permissioned payloads controller will be deployed
+  function run(address controller) public {
```

Comment:
Instead of passing the payloadsController as param here, once we have deployed the new payloadsController, I would add it to address book and import it directly with that and not pass via param

Reply:
Yea, that's exactly what I wanted to mention in the comment above. Will write it more explicitly